src/App.js

```diff
@@ -1,13 +1,36 @@
-
 import './App.css';
 import EmailCardList from "./EmailCardList/EmailCardList.js";
 import Header from './Header/Header.js'
 import SideBar from './SideBar/SideBar.js'
-import {useEffect, useState} from "react"
+import DisplayReadingPanel from "./ReadingPanel";
+import {useState, useEffect} from "react";
 
 const App = () => {
     const [allEmailSnippets, setAllEmailSnippets] = useState([])
+    const [emailSearchId, setEmailSearchId] = useState(98)
+    const [emailDataById, setEmailDataById] = useState([])
+    const [emailRepliesBoolean, setEmailRepliesBoolean] = useState(false)
     const [sidebarIsHidden, setSidebarIsHidden] = useState(true)
+    const [readingPanelDisplay, setReadingPanelDisplay] = useState(false)
+
+    const fetchEmailById = async () => {
+        let queryString = 'http://localhost:8080/emails/' + emailSearchId
```

Comment:
rename this variable to `fetchUrl`

Reply:
also it would be better if both functions `fetchEmailById` and `fetchAllEmailData` were grouped in one place and all the `useEffect`s grouped in one place, so that the file overall is tidier